tests/example/full.py

```diff
@@ -13,13 +16,16 @@
     ],
     'publisher': 'CERN',
     'publicationYear': '2015',
+    'resourceType': {
+        'resourceTypeGeneral': 'Dataset'
+    }
 }
 
 # Validate dictionary
-assert schema31.validate(data)
```

Comment:
In any case, this test is still run here: https://github.com/inveniosoftware/datacite/blob/master/tests/test_schema31.py#L34
But I can create another file for the example. However, except for the `resourceType` that became mandatory in v4.0, there will be no differences between the 2 examples :/